docs/feature-store/feature-sets.md

```diff
@@ -29,13 +29,16 @@ See also [Ingest data using the feature store](#ingest-data-fs)
 
 Create a new {py:class}`~mlrun.feature_store.FeatureSet` with the base definitions:
 
-* **name**&mdash;The feature set name is a unique name within a project. 
-* **entities**&mdash;Each feature set must be associated with one or more index column. When joining feature sets, the entity is used as the key column.
-* **timestamp_key**&mdash;(optional) Used for specifying the time field when joining by time.
-* **engine**&mdash;The processing engine type:
+* **name** &mdash; The feature set name is a unique name within a project. 
+* **entities** &mdash; Each feature set must be associated with one or more index column. When joining feature sets, the entity is used as the key column.
```

Comment:
When joining feature sets, the entity or the relation will be consider as the keys columns.
Also [here](https://github.com/mlrun/mlrun/pull/2540/files#diff-9a7eab2634bd0d70e21b48d754ec7ad00fcb9c851316f5ebc9d914cd22ed48edR12), it a bit misleading

Reply:
@davesh0812 do you mean "key column" ?
"Also here, it a bit misleading" - what exactly?

Reply:
My intention is that today there are two ways to perform the join, so it is important to note that one of the ways to perform the join is by comparing the entities of the featuresets, and the other way is through the relations that have been defined in the featuresets